lab-raziyeh/app/scss/lib/_theme.scss

```diff
@@ -0,0 +1,14 @@
+$app-primary: #a7f7fa;
+$app-secondary: #7fa7af;
+$app-disabled: #abeaeb;
+$app-active: #fa7fa7;
+$app-success: #a7ff7a;
+$app-error: #ff7a7a;
+$app-warn: #ffa77a;
+$app-info: #7aa7ff;
+$app-white: #fff;
+$app-black: #000;
+
+.navbar-nav{
+  margin-top:5px;
+}
```

Comment:
@Raziyehbazargan this shouldn't be in this file - it needs to be in your navbar component SCSS file